broker/lib/fabrik/Agent.js

```diff
@@ -187,18 +187,20 @@ class Agent extends HttpClient {
       .then(ip => this.post(ip, 'lifecycle/preupdate', context, 200, agentCredsBeforeUpdate));
   }
 
-  createCredentials(ips, parameters) {
+  createCredentials(ips, parameters, preBindResponse) {
     const body = {
-      parameters: parameters
+      parameters: parameters,
+      preBindHook: preBindResponse
```

Comment:
can the attrib `preBindHook` be renamed to `actionResponse` or `actions` as this will be inline with this attrib used at other place, [for ex. here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/lib/fabrik/DirectorManager.js#L663)